docs/user-guide/cli-development-roadmap-next.md

```diff
@@ -0,0 +1,9 @@
+# Zowe CLI Development Roadmap (Technical Preview)
+
+<Badge text="Technical Preview"/> The Zowe CLI @next release is a technical preview. Technical previews are for testing only and are not ready for production. Your feedback is valued and appreciated.
+
+To try out technical preview features, [install the @next release of Zowe CLI](cli-install-cli-next.md). This version may receive breaking changes and is intended to gather early feedback on what may become a future LTS release.
+
+The @next version features:
+* Global Profile Configuration - Simplifies profile management by editing, storing, and sharing mainframe configuration details in one location. Use a text editor to quickly populate configuration files with connection details for your mainframe services. For information, see [Using Global Profile Configuration](cli-using-global-profile-configuration.md).
```

Comment:
I would simplify this headline to - improves the initial setup of the command line interface by making configuration information (e.g. service connection details) easier to share and easier to store within projects.